Cargo.toml

```diff
@@ -18,14 +18,14 @@ bitcoin = { version = "0.29.2", features = ["rand"] }
 clap = { version = "4.4.6", features = ["derive"] }
 futures = "0.3.26"
 home = "0.5.5"
-lightning = "0.0.118"
+lightning = { git = "https://github.com/orbitalturtle/rust-lightning", branch = "signature-data-enum" }
 rand_chacha = "0.3.1"
 rand_core = "0.6.4"
 log = "0.4.17"
 log4rs = { version = "1.2.0", features = ["file_appender"] }
 tokio = { version = "1.25.0", features = ["rt", "rt-multi-thread"] }
 tonic = "0.8.3"
-tonic_lnd = { git = "https://github.com/Kixunil/tonic_lnd", rev = "fac4a67a8d4951d62fc020d61d38628c0064e6df" }
+tonic_lnd = { git = "https://github.com/orbitalturtle/tonic_lnd", branch = "update-signer-client" }
```

Comment:
We may want to consider switching over to https://github.com/fedimint/tonic_lnd, they're more actively maintaining the fork (with the plan to upstream their changes if the main repo comes back online). I'm sure they'd be happy to review some PRs.

Reply:
Great idea, looks like they also support a bunch more of the lnd subservers.

Reply:
Looks like they feature gate the subservices, which makes the code slightly different. I'll push up a PR there sometime this week. :)